src/types.ts

```diff
@@ -405,11 +332,7 @@ export type TEvents = {
     ce_questionnaire_form: QuestionnaireForm
     ce_partner_account_signup_form: PartnerAccountSignupForm
     ce_virtual_signup_email_confirmation: VirtualSignupEmailConfirmation
-    ce_bot_dashboard_form: BotDashboardForm
-    ce_bot_quick_strategy_form: BotQuickStrategyForm
-    ce_bot_builder_form: BotBuilderForm
-    ce_bot_tutorial_form: BotTutorialForm
-    ce_contracts_set_up_form: ContractsSetupForm
```

Comment:
don't we need to remove `ContractsSetupForm` as well?

Reply:
We need this. Adding it back. @ali-hosseini-deriv